### DISCUSSION TITLE
How is the musl binary built?

### DISCUSSION THREAD

**Discussion by 1oglop1** [mihaigalos/aim#270] [Q&A] [answered]
Hi, I am learning Rust and I needed statically linked binary which ended up being quite a journey because I do not have any C++ or Go experience. 

I ended up using https://github.com/rust-cross/rust-musl-cross which worked by sheer luck. 
So I am curious to learn a bit more but your github workflow is quite complex to decode.

Would you mind pointing me to some articles, docs or other tips, so that I can learn more about this topic?
Thank you

**Comment by mihaigalos** [comment, marked as answer]
Hi,

Please see `.github/workdlows/cd.yaml` - it's responsible for releasing aim. It calls a reusable workflow from [here](https://github.com/mihaigalos/workflows/blob/main/.github/workflows/rust-cd.yaml).
